### PR TITLE
Skin Feature 2

### PR DESCRIPTION
**Pull request by jonomnom**
Add Vaporware arcade skin. Continuation of https://github.com/OmakaseaNFT/love-game/pull/26 

@GooeyTuxedo

### CODE REVIEW

File: components/randomdialog.tsx
Comment (GooeyTuxedo) [inline components/randomdialog.tsx:1]: we're not using this component anymore so this file can be removed

File: components/filetheme.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+  const closePayment = () => {
+    setShowPayment(false);
+  };
+
+  return (
+    <div className="px-6">
+      {showPayment && (
+        <div className="z-20">
+          <Dialog
```
Comment (GooeyTuxedo) [inline components/filetheme.tsx:33]: looks like `id` is an expected property on this component

**Review by GooeyTuxedo — COMMENTED** [review]
this is some good work!
once we get it building properly, we'll see how it looks on a preview branch

File: components/randomdialog.tsx
Comment (jonomnom) [reply components/randomdialog.tsx:1]: Done.

File: components/filetheme.tsx (hunk 1 above)
Comment (jonomnom) [reply components/filetheme.tsx:33]: Done.

**Comment by jonomnom** [comment]
> this is some good work! once we get it building properly, we'll see how it looks on a preview branch

I am able to build production locally. Could you show the failing build logs?

**Comment by GooeyTuxedo** [comment]
Your last commit took care of the build failure.  :+1: 
We've got a branch preview up now.  In the morning I'll point a testnet heartbreaker instance at the preview so we can verify the game still runs well.

Give the team a day or two to get some eyes on it and we'll drop any more feedback in here :vulcan_salute:

**Comment by GooeyTuxedo** [comment]
@jonomnom @jhinnbay

Hey guys,
we recently added a discord icon to the main screen
did you want to add a themed one to the vaporwave theme?

whenever I get this merged into `main`, we're going to set the vaporwave theme as the default theme so users can see your work featured right away

**Comment by GooeyTuxedo** [comment]
@jonomnom 
Can you verify that the theme saves properly to localstorage?
on the previously built [branch preview](https://love-game-git-fork-jhinnbay-skin-merge-omakasea.vercel.app/) it seems to forget the selected theme on refresh.

edit: ahh, now I see. reading from storage is just [commented out](https://github.com/jhinnbay/Love.GameArcadeSkin/blob/d912eed205c27e4f47eff5f80bae8eda75cf6029/system/context/FileThemeContext.tsx#L62).  no problem

**Comment by jonomnom** [comment]
Resolved in https://github.com/OmakaseaNFT/love-game/pull/49

**Pull request closed without merging**